app/javascript/packs/components/ScrollToTopOnMount.jsx

```diff
@@ -1,11 +1,13 @@
-import React, { useEffect } from 'react';
+import React, { useEffect } from "react";
 
 const ScrollToTopOnMount = () => {
   useEffect(() => {
-    window.scrollTo(0, 0);
+    setTimeout(() => {
+      window.scrollTo(0, 0);
+    });
```

Comment:
setTimeout doesn't need a second param of milliseconds?

Reply:
I don't think so. It defaults to zero.